### PR TITLE
Task/148 - Improve in memory storage for organizations

### PR DESCRIPTION
**Pull request by ghost**
# Converted List of organizations to HashMap

The list of organizations will now be stored in-memory in a hashmap for more efficient access

## Issue #148

### CODE REVIEW

File: app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java [hunk 1]
```diff
@@ -28,23 +29,30 @@ public static OrganizationsSettings getInstance() {
...
-        this.organizations =
+        List<Organization> organizationList =
                 formatter.convertYamlToObject(
                         organizationsFileString, new TypeReference<List<Organization>>() {});
+
+        organizations =
+                organizationList.stream()
+                        .collect(
+                                Collectors.toMap(
+                                        Organization::getName, organization -> organization));
```
Comment (halprin) [inline app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java:46]: I think you can do something with `Identity`?  It's been a while.

```suggestion
                                        Organization::getName, Identity::identity));
```

**Review by halprin — APPROVED** [review]
This looks great!  Good job!

File: app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java (hunk 1 above)
Comment (tjohnson7021) [reply app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java:46]: @halprin @basiliskus I'm not seeing an import for `Identity`, and the IDE is telling me there is no local variable? Where is `Identity::identity` coming from? 
I added screenshot below for reference, I haven't tried to push anything yet.
Screenshot 2023-06-01 at 8 12 29 PM

Comment (halprin) [reply app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java:46]: @tjohnson7021, sorry, I mis-remembered.  It's `Function.identity()`.  It's the equivalent to the `something -> something` Java lambda.

**Pull request merged**